cypress/integration/journalistCanCreateArticle.feature.js

```diff
@@ -0,0 +1,34 @@
+describe("Journalist can create an article", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "POST",
+      url: "http://localhost:3000/api/articles*",
+      response: "fixture:success_message.json",
+    });
+    cy.visit("/");
+  });
+
+  it("successfully with title and body", () => {
+    cy.get("input#title").type("This is the title");
+    cy.get("textarea#body").type(
+      "This is the body this is the body this is the body this is the body this is the body."
+    );
+    cy.get("#post").click();
+    cy.get("#message").should("contain", "Article successfully created!");
+  });
+
+  it("unsuccessfully without entering any title", () => {
+    cy.get("textarea#body").type(
+      "This is the body this is the body this is the body this is the body this is the body."
+    );
+    cy.get("#post").click();
+    cy.get("#message").should("not.be.visible");
```

Comment:
You need to test for a error message here.

Reply:
The input fields have the 'required' property, so we can't send off an incomplete request. Should we change that?